migrations/versions/4159b05e9d23_create_invoice_model.py

```diff
@@ -0,0 +1,58 @@
+"""create Invoice model
+
+Revision ID: 4159b05e9d23
+Revises: 5c4788eea43e
+Create Date: 2024-09-11 14:24:27.544744
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "4159b05e9d23"
+down_revision = "5c4788eea43e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Was this or any of the other migrations in the PR manually edited or were they all auto-generated?